src/plugin.js

```diff
@@ -15,13 +17,10 @@ class Plugin {
     this.logger = context.logger
 
     const { require } = this.context
-    const { Promise, promisify } = require('bluebird')
+    const { Promise } = require('bluebird')
+
+    this.zip = promisify(zip.zip)
```

Comment:
We can do this just once when the file is loaded, instead of during the plugin instantiation. I.e., `const zip = promisify(require('cross-zip').zip)` up at the top. This used to be here only because the Bluebird's promisify was used, which needed to be required using the the context's `require` method.

Reply:
Changed